### PR TITLE
Fix min-width issue at mobile size

### PR DESCRIPTION
**Pull request by michaelwiktorek**

### CODE REVIEW

**Comment by brianta** [comment]
LGTM

File: src/components/Page.js [hunk 1]
```diff
@@ -20,20 +19,17 @@ const Component = ({ children, classes }) => {
...
                 style={{
                     display: "flex",
                     justifyContent: "center",
-                    border: "1px solid blue",
                 }}
             >
                 <div
                     style={{
-                        maxWidth: "100%",
+                        width: "100%",
```
Comment (michaelwiktorek) [inline src/components/Page.js]: If the parent gets too small, this div will never be _wider_ than it.

File: src/pages/index.js [hunk 2]
```diff
@@ -84,7 +84,11 @@ const Home = (props) => {
                 </Typography>
             </div>
             <div
-                style={{ margin: "auto", maxWidth: 1168 }}
+                style={{
+                    margin: "auto",
+                    width: "100%",
+                    maxWidth: 790,
```
Comment (michaelwiktorek) [inline src/pages/index.js]: Dropping the 70% and using an approximation of the observed width. This is the size of the middle column of the site that contains the cards.

File: src/components/Page.js [hunk 3]
```diff
@@ -20,20 +19,17 @@ const Component = ({ children, classes }) => {
             <Style>{`
       body {background: #eee}
     `}</Style>
-            {/* <Grid alignItems="stretch" container justify="center" spacing={0}>
```
Comment (michaelwiktorek) [inline src/components/Page.js]: You could potentially bring the grid back, but the inner grid will need to have `width: 100%`

**Pull request merged**